### ISSUE TITLE
Each article should be displayed only once

### ISSUE THREAD

**Issue by lera777** [BinaryStudioAcademy/bsa-2023-writorium#207]
- [x] The article draft must have a "draft" label. 
- [x] Each article in the "My articles" tab should be displayed only once. 
- [x] If a user publishes an article that was marked as a draft, the "draft" label disappears, and the publication date is displayed.
New task for [Make “Save draft”/ "Publish" button disabled, while the system is in the process of saving.#274](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/issues/274) 

(Currently, both the published article and its draft are displayed.)

![Image](https://user-images.githubusercontent.com/19595678/267300127-1df5d4b3-f93b-4ca8-b7d0-de8ce360cb6d.png)
